converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Worker/FhirTransaction/ImagingStudy/ImagingStudySeriesPropertySynchronizer.cs

```diff
@@ -38,6 +38,9 @@ public void Synchronize(FhirTransactionContext context, ImagingStudy.SeriesCompo
 
             // Add startedElement to series
             AddStartedElement(series, dataset, context.UtcDateTimeOffset);
+
+            // Add numberOfInstances to series
+            AddNumberOfInstances(series, dataset);
```

Comment:
I have approved the pr, can you make sure you update the docs to include the additional fields that are synced.